.gitignore

```diff
@@ -104,6 +104,7 @@ celerybeat.pid
 # Environments
 .env
 .venv
+.venv-*
```

Comment:
don't think .venv is used in this project.

Reply:
Yes, for development, it is.  It's not required, however.  It's not needed for testing.

I normally use only `.venv`, but in this case I ended up having `.venv-3_10` and `.venv-3_11` as I was testing an upgrade from Python 3.10 → 3.11.  Since Python 3.11 is working great, I can get rid of the versioned names and return to `.venv`.

Reply:
@zqian, I've updated the file.  Please resolve this conversation if you think this has been handled satisfactorily.